backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java

```diff
@@ -34,5 +34,31 @@ void WhenListIsEmptyReturnEmptyList() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @DirtiesContext
+    @Test
+    void WhenEntryIsAddedReturnAddedEntry() throws Exception {
+        //When
+        mockMvc.perform(
+                MockMvcRequestBuilders.post("/api/entries")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("{\n" +
+                                "    \"title\": \"testTitle\",\n" +
+                                "    \"description\": \"testDescription\",\n" +
+                                "    \"date\": \"2023-12-03\",\n" +
+                                "    \"amount\": 34,\n" +
+                                "    \"category\": \"INCOME\",\n" +
+                                "    \"interval\": \"MONTHLY\"\n" +
+                                "}")
+        )
+        //Then
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.title").value("testTitle"))
+                .andExpect(jsonPath("$.description").value("testDescription"))
+                .andExpect(jsonPath("$.date").value("2023-12-03"))
+                .andExpect(jsonPath("$.amount").value(34))
+                .andExpect(jsonPath("$.category").value("INCOME"))
+                .andExpect(jsonPath("$.interval").value("MONTHLY"));
+    }
+
```

Comment:
if you like you can also add this line to your test to check that you're getting the expected status(200):

.andExpect(MockMvcResultMatchers.status().isOk())

Reply:
Oh yes i will add that too